portal/tests/pageObjects/portal/materials_page.py

```diff
@@ -47,3 +47,7 @@ def __init__(self, browser):
     def click_pdf_link(self):
         self.browser.find_element_by_id('pdf_link').click()
         return pdf_viewer_page.PDFViewerPage(self.browser)
+
+    def click_ks_link(self, keystage):
```

Comment:
I think `click_keystage_link` might be more readable here

Reply:
+1